pkg/detector/policy.go

```diff
@@ -272,9 +272,13 @@ func (d *ResourceDetector) removeResourceLabelsIfNotMatch(objectReference workv1
 		return false, nil
 	}
 
+	object = object.DeepCopy()
+	objLabels := object.GetLabels()
 	for _, labelKey := range labelKeys {
-		util.RemoveLabel(object, labelKey)
```

Comment:
Just a question, why not use the `util.RemoveLabel` function?

Reply:
improve execution efficiency. every call of `util.RemoveLabel` will get and set labels.

Reply:
Can we extend this function? Like this: `util.RemoveLabels(object, labelKeys...)`

Reply:
sure